### PR TITLE
sys-kernel/dracut: 101 revbump for systemd 255 incompatibility

### PR DESCRIPTION
**Pull request by LaszloGombos**
dracutdevs/dracut --> dracut-ng/dracut-ng

Thank you for your consideration !

CC @AndrewAmmerlaan

### CODE REVIEW

**Review by AndrewAmmerlaan — APPROVED** [review]
Thanks, looks good to me :+1:.

We should also adjust `metadata.xml` with the new upstream:
```
	<upstream>
		<remote-id type="github">dracutdevs/dracut</remote-id>
	</upstream>
```

**Comment by AndrewAmmerlaan** [comment]
> Thanks @AndrewAmmerlaan . Seems there is a build failure that i need some help with, please.
> 
> `distfile missing from Manifest: [ dracut-100.tar.gz ]`

This just means that the entry in the manifest for the new version is missing. It's resolved by regenerating the manifest with `pkgdev manifest`.

File: sys-kernel/dracut/dracut-9999.ebuild [hunk 1]
```diff
@@ -9,23 +9,17 @@ if [[ ${PV} == 9999 ]] ; then
 	inherit git-r3
 	EGIT_REPO_URI="https://github.com/dracutdevs/dracut"
```
Comment (Dessa) [inline sys-kernel/dracut/dracut-9999.ebuild:10]:
```suggestion
	EGIT_REPO_URI="https://github.com/dracut-ng/dracut-ng"
```

**Comment by floppym** [comment]
I think this should be added as dracut-100 (no -r1).

File: sys-kernel/dracut/dracut-100.ebuild [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/dracut-ng/dracut-ng"
+else
+	if [[ "${PV}" != *_rc* ]]; then
+		KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~hppa ~ia64 ~loong ~mips ~ppc ~ppc64 ~riscv ~sparc ~x86"
+	fi
+	SRC_URI="https://github.com/dracut-ng/dracut-ng/archive/refs/tags/${PV}.tar.gz -> ${P}.tar.gz"
+fi
```
Comment (Dessa) [inline sys-kernel/dracut/dracut-100.ebuild:16]: needs
``	S="${WORKDIR}/${PN}-ng-${PV}"``
after SRC_URI

**Review by Dessa — APPROVED** [review]
builds fine now, generated image works too

File: sys-kernel/dracut/Manifest [hunk 3]
```diff
@@ -1,3 +1,4 @@
...
 DIST dracut-060_pre20231030.tar.gz 498954 BLAKE2B ce47f06e1304f732cbad66046c70cbcb03d33d69b9c3e87a4e2e2d30fa837b3b36949462cec9b06125e824ad82db45bd7516c22a1c84ec6884790b37e9c65ab1 SHA512 0cd2fefcd624758063c77836989260a3d5b0d533004c2395124c8200c31602f5d7d0b8c9491de37ce4afb2c5009b2cf944ddd3bb9e0e11120d84176cb1e7423c
 DIST dracut-060_pre20240104.tar.gz 499965 BLAKE2B 935e0e5da348426d69c6dab6b91078f126cadd9ffc6a32378e79cd93b1dbadff35899efc3786fc12bf5a6741843d3637b0c98cc71fe4a96a8caf053ae887bac3 SHA512 f7818265f082e9c05ebb81a91b67fb9b1d3bf8b2433b7e6ea9be6bee43d28cd1ee48577648e1d9b3729c17608b028d294c13bf5d4db4cc5a18e3b007eb2cd67e
+DIST dracut-100.tar.gz 556103 BLAKE2B 639855491ac7800a1ad35f97b87dde637c4e1a0e4de2b644277df38c9bf7a9b179ad595de2e8a7cd580c51b8b953f995c16de9a5b9728120c9bf5f79b7a305ed SHA512 260c0c5761f9a9b12488a0a01d16cac3f832d33a0c896fa7c7e5e5f82f1a753dde78ba779e2d6575184a6efdfb31c5672a0f3c786e98a146308e440ad775565b
```
Comment (Dessa) [inline sys-kernel/dracut/Manifest:4]: ``pkgdev manifest``

**Review by Dessa — CHANGES_REQUESTED** [review]

**Review by AndrewAmmerlaan — APPROVED** [review]

File: sys-kernel/dracut/dracut-101.ebuild [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+	>=app-text/docbook-xsl-stylesheets-1.75.2
+	>=dev-libs/libxslt-1.1.26
+	virtual/pkgconfig
+"
+
+QA_MULTILIB_PATHS="usr/lib/dracut/.*"
+
+PATCHES=(
+	"${FILESDIR}"/gentoo-ldconfig-paths-r1.patch
+	"${FILESDIR}"/dracut-060-grub-layout.patch
```
Comment (AndrewAmmerlaan) [inline sys-kernel/dracut/dracut-101.ebuild:67]: This patch should become unnecessary after merging https://github.com/dracut-ng/dracut-ng/pull/111 upstream.

**Review by floppym — APPROVED** [review]

**Comment by AndrewAmmerlaan** [comment]
Thanks :+1:

**Pull request closed without merging**